eo-maven-plugin/src/main/java/org/eolang/maven/PullMojo.java

```diff
@@ -64,24 +66,17 @@ public final class PullMojo extends SafeMojo {
      * The Git hash to pull objects from, in objectionary.
      *
      * @since 0.21.0
+     * @checkstyle VisibilityModifierCheck (6 lines)
```

Comment:
@maxonfjvipon Could you please explain why we need to exclude the `VisibilityModifierCheck` here?

Reply:
@Graur there's no need for it, thanks